embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -89,5 +89,5 @@ else
     java_args="-XX:+AggressiveOpts -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xverify:none $java_args"
 fi
 
-exec java $java_args -jar "$0" $jruby_args "$embulk_args"
+exec java $java_args -jar "$0" $jruby_args $embulk_args
```

Comment:
The reason why I removed `"` from `$0` is just because it is already quoted at https://github.com/embulk/embulk/pull/604/files#diff-73174f18d3b0fc9c849344f928ca50a0R47

```
embulk_args="$@"
```